From: Antonio Larrosa <alarrosa@suse.com>
Subject: Fix build with srt 1.3.4 since gstreamer expects 1.4.x

SRTO_STRICTENC was just renamed to SRTO_ENFORCEDENCRYPTION in 1.4, so revert that.
SRTO_PACKETFILTER was introduced in 1.4.0 so we can't support urls
specifying a value for it in the url.

Index: gst-plugins-bad-1.22.1/ext/srt/gstsrtobject.c
===================================================================
--- gst-plugins-bad-1.22.1.orig/ext/srt/gstsrtobject.c
+++ gst-plugins-bad-1.22.1/ext/srt/gstsrtobject.c
@@ -177,13 +177,13 @@ SrtOption srt_options[] = {
   {"transtype", SRTO_TRANSTYPE, G_TYPE_INT},
   {"kmrefreshrate", SRTO_KMREFRESHRATE, G_TYPE_INT},
   {"kmpreannounce", SRTO_KMPREANNOUNCE, G_TYPE_INT},
-  {"enforcedencryption", SRTO_ENFORCEDENCRYPTION, G_TYPE_BOOLEAN},
+  {"enforcedencryption", SRTO_STRICTENC, G_TYPE_BOOLEAN},
   {"ipv6only", SRTO_IPV6ONLY, G_TYPE_INT},
   {"peeridletimeo", SRTO_PEERIDLETIMEO, G_TYPE_INT},
 #if SRT_VERSION_VALUE >= 0x10402
   {"bindtodevice", SRTO_BINDTODEVICE, G_TYPE_STRING},
 #endif
-  {"packetfilter", SRTO_PACKETFILTER, G_TYPE_STRING},
+  //{"packetfilter", SRTO_PACKETFILTER, G_TYPE_STRING},
   {"retransmitalgo", SRTO_RETRANSMITALGO, G_TYPE_INT},
   {NULL}
 };
